starters/nestjs-bullmq-postgresql/package.json

```diff
@@ -12,6 +12,7 @@
 		"start:dev": "nest start --watch",
 		"start:debug": "nest start --debug --watch",
 		"start:prod": "node dist/main",
+		"start:all": "yarn infrastructure:start && yarn start:dev",
```

Comment:
Do we have a convention for naming the scripts like that? Is `start:all` following that?